File: Haber/csharp haber/Mail.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace csharp_haber
{
    public class Mail : Iletisim
    {
        public void BilgiGonder(Kullanici kullanici, string mesaj){
            this.MailGonder(kullanici.GetMail(), mesaj);
        }

        public void MailGonder(string mail, string mesaj){
            Console.WriteLine(mail + "adresine " + mesaj + " gonderilmistir.");
        }
    }
}

File: Haber/java haber/Mail.java
public class Mail implements Iletisim {
    @Override
    public void bilgiGonder(Kullanici kullanici, String mesaj){
        this.mailGonder(kullanici.GetMail(), mesaj);
    }

    public void mailGonder(String mail, String mesaj){
        System.out.println(mail + " adresine " + mesaj + " gonderilmistir.");
    }
}
